sc/shared/src/test/scala/sigma/LanguageSpecificationV6.scala

```diff
@@ -265,9 +269,13 @@ class LanguageSpecificationV6 extends LanguageSpecificationBase { suite =>
   }
 
   property("Box properties equivalence (new features)") {
-    // TODO v6.0: related to https://github.com/ScorexFoundation/sigmastate-interpreter/issues/416
-    val getReg = newFeature((x: Box) => x.getReg[Int](1).get,
-      "{ (x: Box) => x.getReg[Int](1).get }",
+    // related to https://github.com/ScorexFoundation/sigmastate-interpreter/issues/416
+    def getReg = newFeature((x: Box) => x.getReg[Long](0).get,
+      "{ (x: Box) => x.getReg[Long](0).get }",
+      FuncValue(
+        Array((1, SBox)),
+        OptionGet(ExtractRegisterAs(ValUse(1, SBox), ErgoBox.R0, SOption(SLong)))
+      ),
```

Comment:
The [idea behind getReg](https://github.com/ScorexFoundation/sigmastate-interpreter/issues/416#issuecomment-2063781813) was to have "computable" index, not just constant.
For example the index can come from Context variable and thus the concrete register will not be fixed in the ErgoTree. This will allow code like:
```val x = box.getReg[Int](getVar[Int](varId).get).get```

So I suggest to add additional test with variable register passed as function argument and have a series of cases in verifyCases.
```"{ (x: (Box, Int)) => x._1.getReg[Long](x._2).get }"```

Reply:
`{ (x: (Box, Int)) => x._1.getReg[Long](x._2).get }` is way different from `box.getReg[Int](getVar[Int](varId).get).get` , 
so I've made a test in `BasicOpsSpecification` which is checking exactly context variable usage:

```
{
   val x = SELF.getReg[Long](getVar[Int](1).get).get
   x == SELF.value
}
```
and so the new test can also be used for reference for people asked for computable index.